oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/OAuth2AuthorizationAttributeNames.java

```diff
@@ -56,4 +56,9 @@
 	 */
 	String ACCESS_TOKEN_ATTRIBUTES = OAuth2Authorization.class.getName().concat(".ACCESS_TOKEN_ATTRIBUTES");
 
+	/**
+	 * The name of the attribute used for the metadata of the {@link OAuth2AccessToken}.
+	 */
+	String ACCESS_TOKEN_METADATA = OAuth2Authorization.class.getName().concat(".ACCESS_TOKEN_ATTRIBUTES");
```

Comment:
Please remove as it's not needed. The `OAuth2TokenMetadata` is associated in the `OAuth2Tokens`